tests/functional/contribute/test_signup.py

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ...pages.contribute.signup import ContributeSignUpPage
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_toggle_category_with_areas(base_url, selenium):
+    page = ContributeSignUpPage(base_url, selenium).open()
+    page.select_coding_category()
+    assert page.is_coding_area_displayed
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_toggle_category_without_areas(base_url, selenium):
+    page = ContributeSignUpPage(base_url, selenium).open()
+    page.select_helping_category()
+    assert not page.is_areas_region_displayed
+
+
+def test_successful_sign_up_coding(base_url, selenium):
+    page = ContributeSignUpPage(base_url, selenium).open()
+    page.select_coding_category()
+    assert page.is_coding_area_displayed
```

Comment:
We don't need this duplicate assert, `test_toggle_category_with_areas` will fail if this is False, and the next line will also fail anyway.